enumerables.rb

```diff
@@ -0,0 +1,145 @@
+module Enumerable
+  def my_each
+    count = 0
+    while count < length
+      yield(self[count])
+      count += 1
+    end
+    self
+  end
+
+  def my_each_with_index
+    count = 0
+    while count < length
+      yield(self[count], count)
+      count += 1
+    end
+    self
+  end
+
+  def my_select
+    result = []
+    my_each do |item|
+      result << item if yield(item)
+    end
+    result
+  end
+
+  def my_all?
+    result = true
+    my_each do |item|
+      result = false unless yield(item)
+    end
+    result
+  end
+
+  def my_any?
+    result = false
+    my_each do |item|
+      result = true if yield(item)
+    end
+    result
+  end
```

Comment:
- [ ] should have working `my_any?` identical and returns the same thing as ruby's [`any?`](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-any-3F). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_any.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))

- Your method doesn't consider the possibility when no block or argument is given. It should return true if the enumerable contains at least one truthy value.

- Your method doesn't consider the possibility when a class is given as an argument `(some_enumerable.my_any?(String)).`

- Your method doesn't consider the possibility when a Regex is given as an argument `(some_enumerable.my_any?(/w/)).`

- Your method doesn't consider the possibility when a string is given as an argument `(some_enumerable.my_any?('cow')).`